scylladb/sphinx-scylladb-theme
# Add scylla university live navbar

Closes #151

## How to test this PR

Download the changes and run ``make preview``. If you close the banner but you want to see the banner again, open ``localhost:5500`` in a new tab using the incognito mode.

## Preview

![image](https://user-images.githubusercontent.com/9107969/114392195-547ce800-9b90-11eb-927a-ecf1a87b85ca.png)

## Rolling out the changes

After you review and merge this PR, I'll roll out a new version of the theme. 
For the scylla-docs project, you will have to deploy a new version of the site.
Every other documentation project will display the top navbar automatically on their next documentation build.
